systemd/test/test_id128.py

```diff
@@ -29,7 +29,8 @@ def test_get_machine_app_specific():
     a1 = uuid.uuid1()
     a2 = uuid.uuid1()
 
-    with skip_oserror(errno.ENOSYS):
+    # yikes, python2 doesn't know ENOMEDIUM
```

Comment:
Given the other PR, maybe merge it first and use enomedium here?

Reply:
I want this patch to be backportable without conflicts… This workaround an be dropped later.